code/obj/item.dm

```diff
@@ -105,6 +107,16 @@
 	var/inventory_counter_enabled = 0
 	var/obj/overlay/inventory_counter/inventory_counter = null
 
+	// bunch of knife fluff so we dont keep rebuilding lists every time we frick around with a knife
+	var/no_fluff_spam_pls = 0
+	var/list/cool_grip_adj = list("a sick", "a wicked", "a deadly", "a menacing", "an edgy", "a tacticool", "a sweaty", "an awkward")
+	var/list/cool_grip1 = list("combat", "fightlord", "guerilla", "hidden", "space", "syndie", "double-reverse", "\"triple-dog-dare-ya\"", "stain-buster's")
+	var/list/cool_grip2a = list("blade", "cyber", "street", "assistant", "comedy", "butcher", "edge", "beast", "heck", "crud", "ass")
+	var/list/cool_grip2b = list("master", "slayer", "fighter", "militia", "space", "syndie", "lord", "blaster", "beef", "tyrannosaurus")
+	var/list/wheredWeSeeIt = list("saw the clown do", "saw the captain do", "saw a large dog do", "saw the head of security do",\
+														"saw someone in a red spacesuit do", "saw a floating saw do", "saw on TV",\
+														"saw one of the diner dudes do", "saw just about every assistant do")
+
```

Comment:
Not more stuff defined on /obj/item please. 😭

Reply:
I didn't want to define the lists in the proc that uses them since they are fairly long lists and the proc can be done repeatedly, I wasn't sure if the game needed to rebuild the lists every time the proc fired.

I'll move it to the proc.